<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>268</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Library content</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="OutsideWidget" name="outside_widget">
     <property name="text">
      <string>Warning: text width exceeds page margins!</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Format:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="text_radio">
         <property name="text">
          <string>Plain &amp;text</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">format_group</string>
         </attribute>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="pdf_radio">
         <property name="text">
          <string>&amp;PDF</string>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">format_group</string>
         </attribute>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;Columns:</string>
         </property>
         <property name="buddy">
          <cstring>col_spin</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="col_spin">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>8</number>
         </property>
         <property name="value">
          <number>2</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="vertical_chk">
         <property name="text">
          <string>&amp;Vertical list mode</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>&amp;Banks:</string>
         </property>
         <property name="buddy">
          <cstring>banks_combo</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="MultiCombo" name="banks_combo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::AdjustToContents</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::MinimumExpanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="3" column="0">
      <widget class="QGroupBox" name="pdf_group">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="title">
        <string>PDF options</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Page &amp;size:</string>
            </property>
            <property name="buddy">
             <cstring>format_combo</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="format_combo"/>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>&amp;Orientation:</string>
            </property>
            <property name="buddy">
             <cstring>orientation_combo</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="orientation_combo">
            <item>
             <property name="text">
              <string>Portrait</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Landscape</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="pagebank_chk">
            <property name="text">
             <string>&amp;1 page per bank</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>&amp;Font:</string>
            </property>
            <property name="buddy">
             <cstring>font_combo</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFontComboBox" name="font_combo"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>&amp;Size:</string>
            </property>
            <property name="buddy">
             <cstring>font_size_spin</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="font_size_spin">
            <property name="minimum">
             <number>6</number>
            </property>
            <property name="maximum">
             <number>72</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Spacin&amp;g:</string>
            </property>
            <property name="buddy">
             <cstring>spacing_spin</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spacing_spin">
            <property name="minimum">
             <number>-48</number>
            </property>
            <property name="maximum">
             <number>128</number>
            </property>
            <property name="singleStep">
             <number>8</number>
            </property>
            <property name="value">
             <number>16</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
       <zorder>horizontalLayoutWidget_4</zorder>
       <zorder>horizontalLayoutWidget_4</zorder>
       <zorder>horizontalLayoutWidget_3</zorder>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MultiCombo</class>
   <extends>QComboBox</extends>
   <header>bigglesworth.classes</header>
  </customwidget>
  <customwidget>
   <class>OutsideWidget</class>
   <extends>QLabel</extends>
   <header>bigglesworth.classes</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>242</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="format_group"/>
 </buttongroups>
</ui>
